addons/mail/static/src/components/message_action_list/message_action_list.scss

```diff
@@ -1,11 +1,22 @@
+.o_MessageActionList {
+  font-size: 0.9rem;
+  padding: map-get($spacers, 1) map-get($spacers, 2);
+}
+
 .o_MessageActionList:hover {
-    box-shadow: 0 4px .5rem -.5rem $black;
+  box-shadow: 0 4px .5rem -.5rem $black;
+}
+
+.o_MessageActionList_action {
+  opacity: 0.4;
+  padding: map-get($spacers, 1) map-get($spacers, 2);
```

Comment:
I think you can simply remove these `padding` rules, as inline bootstrap class `p-2` and `p-1` has precedence

Reply:
These css padding rules acutally still have an effect on the padding, and I couldn't reproduce that effect with added `p-3` and `p-1` rules.
So I left that first css padding rule in :)

Reply:
Indeed, the 1st css had no classname equivalent.
But you can make it with bootstrap classnames, here it's equivalent to `py-1 px-2` I think.